app/views/decidim/solutions/admin/solutions/_form.html.erb

```diff
@@ -37,19 +37,59 @@
     </div>
   </div>
 
-  <div class="card-divider">
-    <h2 class="card-title"><%= t(".problem") %></h2>
-  </div>
-
-  <div class="card-section">
-
-    <div class="row column">
-      <%= form.select :decidim_problems_problem_id,
-                      @form.select_problem_collection,
-                      selected: @form.try(:problem) ? [translated_attribute(@form.try(:problem).title), @form.try(:problem).id] : '',
-                      label: t("models.solution.fields.problem", scope: "decidim.solutions.admin"),
-                      include_blank: "" %>
+  <% if current_participatory_space.components.where(manifest_name: "problems").present? %>
+    <div class="card-divider">
+      <h2 class="card-title"><%= t(".problem") %></h2>
+    </div>
+    <div class="card-divider" style="display:none;">
```

Comment:
why `display: none`? why just don't render challenge title?

Reply:
Because if I don't render it and then the problems module is added and it was related to a challenge, then the value is nil and there was no associated challenge and backwards.

Reply:
I understand, but if the module is removed anyway the associated problem won't exist anymore. Also we don't need to render the title hidden, better don't render it at all.